src/formio/components/Button.stories.mdx

```diff
@@ -0,0 +1,38 @@
+import { Canvas, Meta, Story, Props } from '@storybook/addon-docs';
+import { FormioComponent, FormDecorator } from './story-util';
+
+<Meta
+  title="Form.io components/Button"
+  argTypes={{
+    id: {
+      type: { name: 'string' },
+    },
+    label: {
+      type: { name: 'string', required: false },
+    },
+  }}
+  decorators={[FormDecorator]}
+  parameters={{
+    controls: { sort: 'requiredFirst' }
+  }}
+/>
+
+# Button
+
+## Default
+
+export const Template = ({ id = '', label = '', ...args}) => (
+  <FormioComponent component={{
+      key: id,
+      type: 'button',
+      input: true,
+      label,
+}}/>);
+
+<Canvas>
+  <Story name="Default" args={{ id: 'submit', label: 'Verzenden'}}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<Props story="Default" />
```

Comment:
I think we should remove this as it's misleading - we don't use the formio button component(s) (they are not available in the form designer in the backend).

Reply:
So I guess `src/formio/components/templates/button.ejs` is not used at all. Should we even keep it then? Having it in the codebase is also a bit misleading perhaps.

Reply:
I *could* be wrong - it's possible it is used for `multiple: true` components to add another entry and possibly also in the `datagrid` component.

@SilviaAmAm can you confirm this?

Reply:
@Robbert we looked into it, and the buttons for data grid and `multiple` components are hard `<button>` elements in their templates (in `src/formio/templates`, the files `multiValueRow.ejs` and `multiValueTable.ejs`).

So I think we can remove the `Button` component. Let's exclude it from storybook in this PR, and then make a separate PR/commit to remove the component - if we're wrong, then it's easily reverted.